tests/drivers/i2c/i2c_gecko/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
I am not sure if this is a valid test that can be added to zephyr. If it is, the folder "tests/boards" would be the better location.